src/main/kotlin/com/buildit/bookit/v1/booking/dto/Booking.kt

```diff
@@ -21,6 +22,13 @@ data class BookingRequest(
     val end: LocalDateTime?
 )
 
+const val MASKED_STRING = "**********"
+fun maskIfOther(booking: Booking, otherUser: UserPrincipal): Booking =
```

Comment:
maybe `maskSubjectIfOther()`, unless we might mask other properties at some point (?)

Reply:
we might, but dunno what that would be ATM.. okay i can update that ... can always change it when we know more later